pkgs/development/python-modules/hsh/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchPypi, lib, commandlines }:
+
+buildPythonPackage rec {
+  pname = "hsh";
+
+  version = "1.1.0";
+
+  src = fetchPypi {
```

Comment:
Can't it be built from source? The [repo](https://github.com/chrissimpkins/hsh) contains tests.

Reply:
I guess yes, but then there are extra steps to make r-ryantm automate it all

Reply:
Yeah, but it is preferred to have the tests working just in case.

Reply:
I had to package one more lib for this to work